bots/employee-finder/src/SSW.SophieBot/dialogs/GetCurrentEmployeesOnLocationDialog/GetCurrentEmployeesOnLocationDialog.dialog

```diff
@@ -82,7 +82,7 @@
             },
             {
               "property": "turn.endTime",
-              "value": "=dialog.rawDateEntity.values[0].resolution[0].end"
+              "value": "=coalesce(dialog.rawDateEntity.values[0].resolution[0].end, addDays(formatDateTime(dialog.rawDateEntity.values[0].resolution[0].value), 1))"
```

Comment:
What's the purpose of adding this? This is causing the #626 

![image](https://github.com/SSWConsulting/SSW.SophieBot/assets/37203901/8e851f42-cc79-47fe-b897-86d90948b1e1)

Reply:
Datetime entity returned from Luis has different structure for "who was in Brisbane last Friday"(date) and "who was in Brisbane last week"(date range) so I used coalesce to cater for the two scenarios. As for the error here we need to debug and see the query result from Luis